codeflash/optimization/function_optimizer.py

```diff
@@ -1379,10 +1393,10 @@ def establish_original_code_baseline(
                     code_context=code_context,
                 )
             finally:
-                # Remove codeflash capture
-                self.write_code_and_helpers(
-                    self.function_to_optimize_source_code, original_helper_code, self.function_to_optimize.file_path
-                )
+                if not self.function_to_optimize.is_async:
```

Comment:
we probably don't need this condition here. Otherwise the init function won't be reverted back after adding codeflash_capture